src/components/ProjectSettingsTabs/PluginsTab/getFormSchema.ts

```diff
@@ -171,18 +159,12 @@ export const getFormSchema = (
         ...hiddenIf(formData?.buildBaronSettings.useBuildBaron === true),
         "ui:description":
           "Specify the endpoint and secret for a custom webhook to be called when the File Ticket button is clicked on a failing task.",
-        endpoint: {
-          ...placeholderIf(
-            repoData?.projectPluginsSettings?.buildBaronSettings
-              ?.fileTicketWebhook?.endpoint
-          ),
-        },
-        secret: {
-          ...placeholderIf(
-            repoData?.projectPluginsSettings?.buildBaronSettings
-              ?.fileTicketWebhook?.secret
-          ),
-        },
+        endpoint: placeholderIf(
```

Comment:
After this change, it shows (Default from Repo) without any other content. But that is probably it's own bug with the placeholderIf function. 
![image](https://user-images.githubusercontent.com/13104717/152213833-03fd3a34-69e4-4692-ad89-5834e4a8c3b8.png)

Reply:
Actually if I had to guess, the problem is likely that there's no data for this field in the local document for the project's parent repo. Is this something that is guaranteed to be set for a repo? If not, we should only show a placeholder if the repo value is set.

Reply:
yes, that's probably the case